### ISSUE TITLE
"translations must be broken"

### ISSUE THREAD

**Issue by yangchristina** [sdasda7777/ScheduleBlock#5]
Stopped working after I got this:
<img width="1388" alt="Screenshot 2024-08-06 at 11 26 46 PM" src="https://github.com/user-attachments/assets/ecd4e206-1dbe-4396-9b8e-2369c5b37b1d">
(blocking + editing didn't work)

Worked again after uninstalling and reinstalling the extension

**Comment by sdasda7777** [comment]
Thank you for reporting it! Did you lose your data when uninstalling and reinstalling the extension?

**Comment by yangchristina** [comment]
Yes the data reset after reinstalling, I'm assuming to keep the data you need to export your data first, I didn't have much, so I didn't bother doing that.

**Comment by sdasda7777** [comment]
I'm sorry about that. Do you have any recollection what you were doing before it happened?

**Comment by yangchristina** [comment]
No worries, your extension has helped me a lot, so thanks for that!
I was adding a new regex, I believe it was this format `.*websitename\.com.*`, but considering I was able to add this without problems after reinstalling, maybe I misspelled it or something before? Sorry I haven't been able to reproduce it, I'll let you know if I come across it again.

**Comment by sdasda7777** [comment]
Should be fixed in main branch now.
